forge-gui/res/cardsfolder/upcoming/dunbarrow_revivalist.txt

```diff
@@ -7,8 +7,8 @@ T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.S
 SVar:TrigBoon:DB$ Effect | Boon$ True | Duration$ Permanent | Triggers$ ChangeZoneTrig | SubAbility$ DBBargained
 SVar:ChangeZoneTrig:Mode$ ChangesZoneAll | ValidCards$ Creature.YouCtrl | Destination$ Battlefield | TriggerZones$ Command | OneOff$ True | Execute$ TrigChoose | TriggerDescription$ When one or more creatures enter the battlefield under your control, create a Wicked Role token attached to one of them.
 SVar:TrigChoose:DB$ ChooseCard | Defined$ You | Choices$ Card.TriggeredCards | ChoiceTitle$ Choose one of the cards that entered the battlefield | Mandatory$ True | SubAbility$ DBToken
-SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ role_wicked | TokenOwner$ You | AttachedTo$ ChosenCard
-SVar:DBBargained:DB$ ChangeZone | Condition$ Bargain | Origin$ Graveyard | Destination$ Hand | Hidden$ True | ChangeType$ Creature.YouOwn | ChangeNum$ 1
+SVar:DBToken:DB$ Token | TokenScript$ role_wicked | AttachedTo$ ChosenCard
+SVar:DBBargained:DB$ ChangeZone | ConditionPresent$ Card.Self+bargained | Origin$ Graveyard | Destination$ Hand | Hidden$ True | ChangeType$ Creature.YouOwn | ChangeNum$ 1
```

Comment:
this still seems wrong in case it LTB before resolving the trigger
use `ConditionDefined$ Self`